src/compiler/scala/tools/nsc/backend/jvm/BCodeIdiomatic.scala

```diff
@@ -23,7 +23,7 @@ import scala.tools.nsc.backend.icode.Primitives.{NE, EQ, TestOp, ArithmeticOp}
  */
 trait BCodeIdiomatic {
   val int: BackendInterface
-  lazy val bTypes = new BTypesFromSymbols[int.type](int)
+  final lazy val bTypes = new BTypesFromSymbols[int.type](int)
```

Comment:
My guess would be https://github.com/lampepfl/dotty/issues/50 & https://github.com/lampepfl/dotty/pull/1051